src/pages/index.js

```diff
@@ -19,41 +21,132 @@ import {
   addNewCardButton,
   settings,
 } from "../utils/constants.js";
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
+import Api from "../components/API.js";
+import { data } from "autoprefixer";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "50644799-43da-42c8-ae19-a67fa2a948b2",
+    "Content-Type": "application/json",
+  },
+});
 
 const previewImagePopup = new PopupWithImage({
   popupSelector: "#card-preview-modal",
 });
-const addCardPopup = new PopupWithForm("#add-card-modal", (data) => {
-  const cardData = {
-    name: data.title,
-    link: data.url,
-  };
-  const newCard = createCard(cardData);
-  cardList.addItem(newCard);
-  addCardFormElement.reset();
-  addFormValidator.disableButton();
-});
 
 const cardList = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (data) => {
       cardList.addItem(createCard(data));
     },
   },
   ".cards__list"
 );
 
+const confirmDeleteModal = new PopupWithConfirm(
+  "#delete-modal",
+  handleDeleteClick
+);
+confirmDeleteModal.setEventListeners();
+
+const editAvatarModal = new PopupWithForm(
+  "#edit-avatar-modal",
+  handleAvatarEditSubmit
+);
+editAvatarModal.setEventListeners();
+
+const addCardPopup = new PopupWithForm("#add-card-modal", (data) => {
+  const cardData = {
+    name: data.title,
+    link: data.url,
+  };
+  api
+    .addCards(cardData)
+    .then((res) => {
+      const card = createCard(res);
+      cardList.addItem(card);
+      addCardFormElement.reset();
+      addFormValidator.resetValidation();
+      addCardPopup.close();
+    })
+    .catch((err) => {
+      console.error(`Error, could not add card: ${err}`);
+    });
+});
+
 const userInfo = new UserInfo({
   nameSelector: ".profile__title",
   descriptionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
 });
 
-const editFormValidator = new FormValidator(settings, profileEditForm);
-const addFormValidator = new FormValidator(settings, addCardFormElement);
+function handleLikeClick(data) {
+  if (!data._isLiked) {
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.